components/create-collective/index.js

```diff
@@ -46,8 +57,6 @@ class CreateCollective extends Component {
     this.setState({ creating: true });
 
     // prepare object
-
-    collective.tags = [this.props.router.query.category];
```

Comment:
Note that this is where the default tags (example:  `community`) takes effect. My thinking is that this isn't needed anymore as we give the users the ability to create tags and if they don't choose any tags we should leave the tags empty. Let me know if you think otherwise. 🤔 

cc: @Betree

Reply:
Does this mean new collectives won't get the "opensource" tag (for OSC) nor "community" (for foundation)? If so, did you discuss it with OSC/OCF admins to make sure they don't rely on it and are ok to change this?

My personal take is that we want all collectives created under OSC to have the "opensource" tag, they shouldn't have to set it manually.

Reply:
You are correct, I think I got this idea that we need to remove these default tags from a conversation in the search experience redesign project, but I don't think we need to do this right now. Let us keep this default tags for the purpose of this PR and add the once that the user specifies in addition to those. I've made the changes. 👍🏽